adstb/bumblebee.py

```diff
@@ -212,8 +231,39 @@ def build_meta_tags(solrdoc):
         
     if 'doi' in data:
         data['doi'] = data['doi'][0]
+    if 'identifier' in data:
+        # expected format: ["2011arXiv1108.0669H", "2012ApJS..199...26H", "10.1088/0067-0049/199/2/26", "arXiv:1108.0669"]
+        for i in data['identifier']:
+            if i.startswith('arXiv'):
+                data['arxiv_id'] = i
+    if 'keyword' in data:
+        data['keyword'] = ', '.join(data['keyword'])
+    if 'page' in data:
+        data['page'] = data['page'][0]
+    if 'page_range' in data and '-' in data['page_range']:
+        # expected format: 41-59
+        data['lastpage'] = data['page_range'].split('-')[1]
+    if 'title' in data:
+        data['title'] = _html_to_text(data['title'][0])
+    link_url_template = 'http://dev.adsabs.harvard.edu/abs/{}'
+    if 'LINK_URL_TEMPLATE' in conf:
+        data['url'] = conf['LINK_URL_TEMPLATE'].format(data['bibcode'])
```

Comment:
i'm confused by this variable, it is not used in the tags sections; and also, i think it shouldn't be, the template has been generated maybe from a different domain that what's in the config

(which brings up the sensitive point of "should we verify/sanitize links in the template" ?)

Reply:
updated pr